proto/dymension/rollapp/rollapp.proto

```diff
@@ -41,12 +41,14 @@ message Rollapp {
   repeated string permissionedAddresses = 5;
   // tokenMetadata is a list of TokenMetadata that are registered on this rollapp
   repeated TokenMetadata tokenMetadata = 6;
+  // registeredDenoms is a list of registered denoms on this rollapp
+  repeated string registeredDenoms = 7;
```

Comment:
let make this variable `=10`, to avoid compatibility issues
otherwise, store migration needed